db/migrate/20200724005110_add_uid_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUidToUsers < ActiveRecord::Migration[6.0]
```

Comment:
Now that we are getting a literal string of first and last name we will use this to ID users.